api/src/prescription/campaign/application/campaign-route.js

```diff
@@ -56,6 +56,21 @@ const register = async function (server) {
         tags: ['api', 'campaign'],
       },
     },
+    {
+      method: 'GET',
+      path: '/api/campaigns/{campaignId}/tutorial-steps',
+      config: {
+        auth: false,
```

Comment:
**question sécu**
on est sûr que tout le monde puisse accéder à cette route sans authent ? 

parceque là n'importe qui ayant l'appel de ce getter peut récupérer en itérant sur les id ? 🤔

Reply:
Yes, le design veut que la demande de connexion arrive à la fin du didacticiel.
Et ça vaut aussi pour les parcours autonomes

Reply:
Alors peut-être un prehandler qui vérifie que le campaignId est bien celui qui porte les tutorials. pour "protéger" un peu les appel aux usecases ?

Reply:
Mais du coup ça change le worklfow des campagnes :O

Reply:
Parcequ'actuellement la route tutorial est assez loin dans le parcours avec ce genre de beforeModel

`this.session.requireAuthenticationAndApprovedTermsOfService`

les différents cas d'accès aux parcours sont repertorié ici https://miro.com/app/board/uXjVM8w7oGY=/ . Qui seront migré dans confluence .

Reply:
Pour avancer de notre côté (le front sera sous FT) ça va si dans un premier temps on merge sans auth ?

Reply:
L'authentification a été ajoutée